lib/agent0/tests/interactive_hyperdrive_test.py

```diff
@@ -0,0 +1,190 @@
+"""Tests interactive hyperdrive end to end"""
```

Comment:
we're moving to a pattern of putting tests with the source files they are testing, unless they are integrations or systems tests. I think this qualifies as one that should live in the `interactive/` folder, but if you disagree I'm fine with leaving it here.